New Names + Minor_Major Grief Strings

izzyphobic opened the pull request:
- 300+ new loner/kittypet names
- Doubled the amount of minor_major grief strings

izzyphobic committed:
New Names + Minor_Major Grief Strings

- 300+ new loner/kittypet names
- Doubled the amount of minor_major grief strings

scribblecrumb commented on resources/dicts/events/death/death_reactions/minor_major.json:
```diff
@@ -7,7 +7,16 @@
     "Some of the memories shared at m_c's vigil make r_c laugh. Some cry. Most of them do both, as r_c marvels at what a special cat m_c was.",
-    "r_c wonders if, maybe, if they're lucky, m_c might visit them in a vision from StarClan."
+    "r_c wonders if, maybe, if they're lucky, m_c might visit them in a vision from StarClan.",
+    "r_c retires to their nest once the vigil comes to a close, their eyes puffy and watery. They ponder over how m_c won't ever get to rise in the morning again, sniffling and curling up a little tighter.",
```
At first this read as though the dead cat was the one 'sniffling and curling up a little tighter', perhaps reword a little so that the subject of the sentence is clearer?

scribblecrumb commented on resources/dicts/events/death/death_reactions/minor_major.json:
```diff
@@ -7,7 +7,16 @@
+    "r_c retires to their nest once the vigil comes to a close, their eyes puffy and watery. They ponder over how m_c won't ever get to rise in the morning again, sniffling and curling up a little tighter.",
+    "After the vigil, r_c makes it a point to remove scraps of foliage from m_c's nest before it is cleared out. They are comforted by m_c's scent and the strands of fur they left behind as they drift to sleep that night.",
+    "As r_c's clanmates comfort them during the vigil, they can't help but wonder if things will ever feel normal again without m_c in their life.",
+    "r_c rubs their weary, wet eyes, gazing up into the vast sky above. Perhaps m_c has already found their way to Starclan and they'd spot a new star in Silverpelt tonight.",
```
need to capitalize StarClan correctly

scribblecrumb commented on resources/dicts/events/death/death_reactions/minor_major.json:
```diff
@@ -7,7 +7,16 @@
+    "As r_c's clanmates comfort them during the vigil, they can't help but wonder if things will ever feel normal again without m_c in their life.",
+    "r_c rubs their weary, wet eyes, gazing up into the vast sky above. Perhaps m_c has already found their way to Starclan and they'd spot a new star in Silverpelt tonight.",
+    "r_c's body feels heavy as exhaustion pulls them down, their mind foggy with emotion. Even so, after the vigil, they amble around camp sluggishly, hoping that somehow, somewhere they'd find m_c alive, well, and happy.",
+    "r_c knew that m_c might not always be able to be by their side, but it wasn't supposed to be so soon. They visit m_c's grave and recall all of the happy memories they shared, bittersweet tears springing to their eyes.",
```
Might not be good to mention "grave" here as it could easily come after a grief message that references seeing the body or touching the body

izzyphobic replied:
This was presumably after the vigil, and I'd think all cats get a grave, body or not? I tested them all to see how they sounded in sentences with the other strings and it seemed fine. Would a reword work if I put, "in the coming days"?

![image](https://user-images.githubusercontent.com/123095928/223288663-c3196827-3a95-4a05-940c-f7375d0ef50f.png)

scribblecrumb replied:
Mm, fair point, it does sound fine!  You can leave as is, then.

scribblecrumb commented on resources/dicts/events/death/death_reactions/minor_major.json:
```diff
@@ -18,6 +27,15 @@
+    "r_c only retires to their nest once ushered by concerned clanmates, their eyes puffy and watery. They sob quietly, tremors wracking their body when they realize m_c won't ever get to rise in the morning again.",
+    "r_c makes it a point to remove scraps of foliage from m_c's nest, snapping at any hindrance. Their paws tremble as they clutch the material, distraught when they realize just how fast m_c's scent was fading.",
+    "Clanmates are unable to comfort r_c during the vigil; they are absolutely inconsolable. They are convinced that they'll never feel normal again, not without m_c beside them.",
+    "In the dead of the night, r_c wails to the vast, inky sky above. They couldn't fathom how Starclan could be so cruel. r_c glares at Silverpelt, their claws trembling as they leave wounds in the earth beneath them.",
```
capitalize StarClan

scribblecrumb commented on resources/dicts/events/death/death_reactions/minor_major.json:
```diff
@@ -18,6 +27,15 @@
+    "Clanmates are unable to comfort r_c during the vigil; they are absolutely inconsolable. They are convinced that they'll never feel normal again, not without m_c beside them.",
+    "In the dead of the night, r_c wails to the vast, inky sky above. They couldn't fathom how Starclan could be so cruel. r_c glares at Silverpelt, their claws trembling as they leave wounds in the earth beneath them.",
+    "r_c is barely able to move their body, fatigue threatening to consume them. Once night falls, they desperately wail m_c's name, their movements frantic as they struggle - no, refuse to accept their death.",
+    "r_c never considered the possibility that they'd one day be without m_c. They visit their grave and stare blankly at the ground for hours upon hours, seeming like nothing but a husk of their former self.",
```
same grave comment as before

izzyphobic committed:
Merge branch 'development' into Izzyphobic

izzyphobic committed:
New Names + Minor_Major Grief Strings

- 300+ new loner/kittypet names
- Doubled the amount of minor_major grief strings
